src/main.cpp

```diff
@@ -539,14 +539,15 @@ int main(int argc, char* argv[])
     VideoManager->ApplySettings();
 
     // Now the settings are loaded, let's set the windows translated title.
-    /// tr: The window title only supports UTF-8 characters in SDL2.
+    // Translators: The window title only supports UTF-8 characters in SDL2.
```

Comment:
Sorry, this is wrong, as @gunchleoc changed the translator comment system in https://github.com/RyDroid/ValyriaTear/commit/26bbbe145825769a7abf52ab5f1a0032c23d8cff
Could you change it back to tr: ?
Thanks